### PR TITLE
[IMP] Code update for new api

### PR DESCRIPTION
**Pull request by Daniel-CA**
Code update for new api

### CODE REVIEW

File: sale_order_line_description/models/__init__.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+#
+#
+#    Copyright (C) 2013 Agile Business Group sagl
```
Comment (tafaRU) [inline sale_order_line_description/models/__init__.py]: Year is not updated here, please use the header contained in my branch instead.

File: sale_order_line_description/models/sale.py [hunk 2]
```diff
@@ -19,36 +19,31 @@
...
-                    cr, uid,
                     'sale_order_line_description.'
-                    'group_use_product_description_per_so_line',
-                    context=context
-            ):
-                product = self.pool['product.product'].browse(
-                    cr, uid, product, context=context)
-                if product.description_sale:
+                    'group_use_product_description_per_so_line'):
+                product_reg = product_obj.browse(product)
```
Comment (tafaRU) [inline sale_order_line_description/models/sale.py]: What does `product_reg` stand for? I would prefer use `product` only.

**Comment by tafaRU** [comment]
@Daniel-CA thanks a lot for your PR! See my comments inline.

File: sale_order_line_description/models/sale.py (hunk 2 above)
Comment (Daniel-CA) [inline sale_order_line_description/models/sale.py]: I prefered to maintain product as it comes from product_id_change with only id, and make a new var for its registry, but as you wish.

**Pull request merged**